client/src/modules/Course/Components/ReviewModal.tsx

```diff
@@ -66,8 +74,23 @@ const ReviewModal = ({
   }, [professorOptions])
   useEffect(() => {
     setAllowSubmit(valid.professor && valid.major && valid.grade && valid.text)
+    if (isLoggedIn) {getNoReviews()}
   }, [valid])
 
+  /**
+   * Determines if the current user has no reviews, so they should receive
+   * the anonymous modal
+   */
+  async function getNoReviews() {
```

Comment:
I'm wondering if it'd be useful to show the anonymous modal for all users regardless of whether they've left a review or not? Just for in case the user forgets that the reviews are anonymous (or perhaps after a certain period of time).